src/api/handlers/appointments/sendReminders.js

```diff
@@ -20,8 +20,8 @@ export const sendReminder = async (appointment) => {
     appointmentDateTime: moment(appointment.appointmentTime).format(
       "YYYY-MM-DD h:mm a"
     ),
-    appointmentDate: moment(appointment.appointmentTime).format("YYYY-MM-DD"),
-    appointmentTime: moment(appointment.appointmentTime).format("h:mm a"),
+    appointmentDate: moment(appointment.appointmentTime).format("LL"),
```

Comment:
What are `LL` and `LTS`?

Reply:
LTS : 'h:mm:ss A', LT : 'h:mm A', L : 'MM/DD/YYYY', LL : 'MMMM D, YYYY', LLL : 'MMMM D, YYYY h:mm A', LLLL : 'dddd, MMMM D, YYYY h:mm A' 

I changed it before I seen you had already updated these lines. I'm not sure my changes are correct in terms of how it's supposed to display. For example LTS includes seconds, which are irrelevant. LT might be better.